### DISCUSSION TITLE
Welcome to the WiX Toolset Discussions experiment on GitHub

### DISCUSSION THREAD

**Discussion by robmen** [wixtoolset/issues#6278] [Announcements]
## Greetings!

We’re starting an experiment using Discussions as a place to talk about the WiX Toolset. This is a great place to ask your "how" and "why" questions.

If the experiment goes well, we will likely decommission the wix-users mailing list. If the experiment does not go well, we'll probably delete the content here. So, don't write down your life's work here. It may not last.

Thanks for being part of the experiment.

**Comment by joeluebker** [comment]
A searchable discussion list sounds like a good step forward. It is difficult, at least for me, to keep up with the emails from mailing list. Hoping this takes off.

Joe

**Comment by Silvenga** [comment]
Mailing lists are quickly becoming a barrier-to-entry for many new developers. So many conventions and nuances, needing to configure a email client correctly, mime types, etc.

I would love to see this succeed!

**Reply by robmen** [reply]
@Silvenga if this GitHub Discussion forum doesn't work well, we will consider moving to a (very likely Discourse-based) web forum provided by [FireGiant](https://www.firegiant.com/).

**Comment by cpuwzd** [comment]


**Reply by BMurri** [reply]
It would be best to create a new discussion item rather than make a comment on a completely unrelated topic.

The only thing that `OneTimeWixBuildInitialization.proj` ever did (IIRC) is to register the strong-name key because we didn't (in v3) distribute the strong-name private key file.

When you move this to its own discussion topic, please indicate if you are trying to build v3 or v4, as the build instructions and requirements are quite different.

**Reply by robmen** [reply]
Dev conversations still happen on wix-devs maliing list while we experiment with Discussions. I'll delete these messages later since it should have been a new conversation as Blair pointed out.

**Reply by rseanhall** [reply]
This a continuation of the discussion in https://github.com/wixtoolset/issues/discussions/6307. Like Rob said, that discussion really should have been on the [wix-devs mailing list](https://wixtoolset.org/documentation/mailinglist/).
